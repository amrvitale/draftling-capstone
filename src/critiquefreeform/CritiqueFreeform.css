textarea {
    width: 300px;
	height: 120px;
	border: 3px solid #cccccc;
	padding: 5px;
}

textarea {
    width: 300px;
	height: 120px;
	border: 3px solid #cccccc;
	padding: 5px;
}

label {
	font-size: 25px;
	font-weight: bold;
	color: whitesmoke;
}
.freeformHero{
  	background-image: url('./freeformStar.jpg');
    height: 100%; 
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
}

.freeformHeading{
	color: #35322a;    
	font-family: 'Playfair Display', 'Georgia', serif;
	text-shadow: 
	4px 4px 0 rgb(175, 33, 33),
    -1px -1px 0 rgb(175, 33, 33),  
     1px -1px 0 rgb(175, 33, 33),
     -1px 1px 0 rgb(175, 33, 33),
      1px 1px 0 rgb(175, 33, 33);
	
}

.freeformTagline{
	color: black;
	background-color: whitesmoke;
}

