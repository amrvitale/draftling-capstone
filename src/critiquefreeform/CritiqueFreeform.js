import React from 'react';
import Read from '../read/Read';
import './CritiqueFreeform.css'

class CritiqueFreeform extends React.Component {
    render() {
        return (
            <div className="freeform">
                <section className="comments">
                    <h2>Post a Freeform Critique</h2>
                    <p>Please remember to be objective, especially if the piece you're critiquing is not in a style or genre that you prefer. Always start with the strengths of the piece, then make solid suggestions for improvement.</p>
                    <h2>Opening Comments</h2>
                    <textarea>
                        
                    </textarea>

                    <h2>Freeform Critique</h2>
                    <textarea>
                        
                    </textarea>

                    <br/>
                    <input type="submit" value="Post Critique"></input>
                </section>
            </div>
        );
    }
}

export default CritiqueFreeform;