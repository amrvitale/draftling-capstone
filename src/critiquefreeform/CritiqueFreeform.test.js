import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CritiqueFreeform from './CritiqueFreeform';

describe (`CritiqueFreeform component`, () => {
    it('renders a .CritiqueFreeform by default', () => {
        const wrapper = shallow(<CritiqueFreeform />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})