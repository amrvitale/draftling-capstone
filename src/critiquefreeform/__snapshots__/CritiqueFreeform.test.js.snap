// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CritiqueFreeform component renders a .CritiqueFreeform by default 1`] = `
<div
  className="freeform"
>
  <div
    className="freeformHero"
  >
    <h1
      className="freeformHeading"
    >
      Post a Freeform Critique
    </h1>
    <p
      className="freeformTagline"
    >
      Please remember to be objective, especially if the piece you're critiquing is not in a style or genre that you prefer. Always start with the strengths of the piece, then make solid suggestions for improvement.
    </p>
  </div>
  <br />
  <br />
  <form
    onSubmit={[Function]}
  >
    <label>
      Opening Comments
    </label>
    <br />
    <textarea
      name="opening"
      onChange={[Function]}
      style={
        Object {
          "boxSizing": "border-box",
          "fontSize": "15px",
          "minHeight": "380px",
          "minWidth": "380px",
          "padding": "9px",
          "textAlign": "left",
        }
      }
    />
    <br />
    <br />
    <label>
      Freeform Critique
    </label>
    <br />
    <textarea
      name="critfreeform"
      onChange={[Function]}
      required={true}
      style={
        Object {
          "boxSizing": "border-box",
          "fontSize": "15px",
          "minHeight": "380px",
          "minWidth": "380px",
          "padding": "9px",
          "textAlign": "left",
        }
      }
    />
    <br />
    <input
      type="submit"
      value="Post Critique"
    />
  </form>
</div>
`;
